### DISCUSSION TITLE
Upload Issue

### DISCUSSION THREAD

**Discussion by dtquocbao** [oqtane/oqtane.framework#1378] [Q&A]
I'm trying to play around with upload file using FileManager but got failed so I was trying to test it with Swagger and this is what I got

![image](https://user-images.githubusercontent.com/4122092/118851517-b236db00-b886-11eb-8f71-4e8d10f4c8fb.png)

So I put a break point in MergeFile() and figured out that the ".part_" somehow not work so application crashes at that place.

![image](https://user-images.githubusercontent.com/4122092/118851768-f7f3a380-b886-11eb-9117-134155232443.png)

Any idea?

Thanks

**Comment by sbwalker** [comment]
Please provide the steps to reproduce.

**Comment by sbwalker** [comment]
The FileController upload method expects the posted file to be split into chunks with a specific naming convention which are then reassembled on the server. If you want to see how this works under the covers look in the interop.js file and uploadFiles() method. You will see that breaks file into "parts". So it you want to test this using Swagger you will need to emulate the behavior of uploadFiles() and use the same naming convention for your file ( ie. use a file extension with  .part_001_001 ).
